ccd-definition/CaseEvent/User/UserEvents-caseworkerEvents-nonprod.json

```diff
@@ -11,7 +11,9 @@
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/submitted",
     "ShowSummary": "N",
     "ShowEventNotes": "N",
-    "RetriesTimeoutURLAboutToSubmitEvent": 0
+    "RetriesTimeoutURLAboutToSubmitEvent": 0,
+    "Publish": "Y"
+
```

Comment:
Remove this blank line if you get the chance please.

Reply:
Removed @ss-evoco